### PR TITLE
fix: Notification appName is not translated

### PR DESCRIPTION
**Pull request by yixinshark**
as title

Log: as title
pms: BUG-294157

### CODE REVIEW

File: panels/notification/server/notificationmanager.cpp [hunk 1]
```diff
@@ -454,6 +457,14 @@ QString NotificationManager::appIdByAppName(const QString &appName) const
         if (item.id == appName || item.appName == appName)
             return item.id;
     }
+
+    // TODO temporary fix for AppNamesMap
+    QScopedPointer<DConfig> config(DConfig::create("org.deepin.dde.shell", "org.deepin.dde.shell.notification"));
```
Comment (18202781743) [inline panels/notification/server/notificationmanager.cpp:462]: 这个加个缓存吧，只有加载时读一下就好了，

File: panels/notification/server/notificationmanager.cpp [hunk 2]
```diff
@@ -192,13 +192,16 @@ uint NotificationManager::Notify(const QString &appName, uint replacesId, const
         return 0;
     }
 
+    QString tsAppName = m_setting->appValue(appId, NotificationSetting::AppName).toString();
+    qInfo(notifyLog) << "ts appName" << tsAppName << ", AppId" << appId;
```
Comment (18202781743) [inline panels/notification/server/notificationmanager.cpp:196]: 日志不加吧，或者换成debug的，

File: panels/notification/server/notificationmanager.cpp (hunk 1 above)
Comment (yixinshark) [reply panels/notification/server/notificationmanager.cpp]: done

File: panels/notification/server/notificationmanager.cpp (hunk 2 above)
Comment (yixinshark) [reply panels/notification/server/notificationmanager.cpp]: done

**Review by 18202781743 — APPROVED** [review]

**Pull request merged**